src/entity-configuration/definitions/fields-defs/enums.ts

```diff
@@ -50,7 +50,7 @@ export enum EntityCoreFields {
   EModelResponse = 'eModelResponse',
   MEModelMorphologyPreview = 'meModelMorphologyPreview',
   MEModelTracePreview = 'meModelTracePreview',
-  MEModelValidated = 'meModelValidated',
+  MEModelValidated = 'validation_status',
```

Comment:
Maybe 
```suggestion
  ValidationStatus = 'validation_status',
```

Just to be consistent (I don't know for sure if that makes sense because this enum is an aggregation of all fields)

Reply:
Or MEModelValidationStatus

Reply:
fixed